mocks/flask-react-mocks/server/server.py

```diff
@@ -0,0 +1,54 @@
+from flask import Flask
+from flask import jsonify
+from flask import request
+from flask import make_response
+from sampleData import *
+from constants import CONSTANTS
+import ast
+import datetime
+app = Flask(__name__)
+
+# List Endpoints
+@app.route(CONSTANTS['ENDPOINT']['LIST'])
+def getList():
+    return jsonify(
+        sampleData['listTextAssets']
+    )
+
+@app.route(CONSTANTS['ENDPOINT']['LIST'], methods = ['POST'])
+def addListItem():
+    data = request.data.decode('utf-8')
+    data = ast.literal_eval(data)
+    listItem = {'_id': sampleData['listId'], 'text': data['text']}
+    sampleData['listTextAssets'].append(listItem)
```

Comment:
Feel like you should add items like so: sampleData['listTextAssets'] = [listItem, *sampleData['listTextAssets']]
or even an sampleData['listTextAssets'].insert(0, listItem) would be good because of the following bug:

added items and saw this Before refreshing: 
![image](https://user-images.githubusercontent.com/32103099/58599988-84e36380-8250-11e9-9d74-b13e855c3fe5.png)

After refreshing:
![image](https://user-images.githubusercontent.com/32103099/58600012-a5abb900-8250-11e9-9d14-f84676c3ef60.png)